build/rewrite.go

```diff
@@ -146,8 +193,7 @@ func keepSorted(x Expr) bool {
 // Second, it removes redundant target qualifiers, turning labels like
 // "//third_party/m4:m4" into "//third_party/m4" as well as ones like
 // "@foo//:foo" into "@foo".
-//
-func fixLabels(f *File) {
+func fixLabels(w *Rewriter, f *File) {
```

Comment:
The *Rewriter object is not used for almost all the functions, please move it to the last place of the arguments list and rename to `_` for the functions where it's not used.

Reply:
Keeping it as the intended use is to use rewriter which is passed in, by default it will contain table json properties

Reply:
How does it prevent moving it to the last place and renaming to `_` in the functions that don't use it?